packages/trader/src/Assets/Trading/Categories/trade-categories.tsx

```diff
@@ -1,21 +1,36 @@
 import React from 'react';
 import { Text } from '@deriv/components';
-import { VANILLALONG, TURBOS } from '@deriv/shared';
-import { localize, Localize } from '@deriv/translations';
+import { Localize } from '@deriv/translations';
 import AccumulatorTradeDescription from './Description/accumulator-trade-description';
-import TurbosTradeDescription from './Description/turbos-trade-description';
+import AsianTradeDescription from './Description/asian-trade-description';
+import CallPutSpreadTradeDescription from './Description/call-put-spread-trade-description';
+import EndTradeDescription from './Description/end-trade-description';
+import EvenOddTradeDescription from './Description/even-odd-trade-description';
+import HighLowTradeDescription from './Description/high-low-trade-description';
+import LbHighLowTradeDescription from './Description/lb-high-low-trade-description';
+import LbPutTradeDescription from './Description/lb-put-trade-description';
+import LbCallTradeDescription from './Description/lb-call-trade-description';
+import MatchDiffTradeDescription from './Description/match-diff-trade-description';
 import MultiplierTradeDescription from './Description/multiplier-trade-description';
+import OverUnderTradeDescription from './Description/over-under-trade-description';
+import RiseFallTradeDescription from './Description/rise-fall-trade-description';
+import RunHighLowTradeDescription from './Description/run-high-low-trade-description';
+import ResetTradeDescription from './Description/reset-trade-description';
+import StayTradeDescription from './Description/stay-trade-description';
+import TurbosTradeDescription from './Description/turbos-trade-description';
+import TouchTradeDescription from './Description/touch-trade-description';
+import TickHighLowTradeDescription from './Description/tick-high-low-trade-description';
+import VanillaTradeDescription from './Description/vanilla-trade-description';
+import { VANILLALONG, TURBOS } from '@deriv/shared';
```

Comment:
@kate-deriv let's not move this line down here and leave it on the top as on master :)